Rakefile

```diff
@@ -4,6 +4,9 @@
 require 'bundler'
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+
+PROJECT_DIR = File.expand_path(".")
```

Comment:
There's some history as to why the Rakefiles are separate, so I'm fine with this as a temporary hack until we have a reason to diverge.